runtime/near-vm-logic/src/gas_counter.rs

```diff
@@ -1,12 +1,24 @@
 use crate::config::{ActionCosts, ExtCosts, ExtCostsConfig};
 use crate::types::{Gas, ProfileData};
 use crate::{HostError, VMLogicError};
-use near_runtime_fees::Fee;
+use near_runtime_fees::{EvmGas, Fee};
 
 #[cfg(feature = "costs_counting")]
 thread_local! {
     pub static EXT_COSTS_COUNTER: std::cell::RefCell<std::collections::HashMap<ExtCosts, u64>> =
         Default::default();
+
+    pub static EVM_GAS_COUNTER: std::cell::RefCell<EvmGas> = Default::default();
+}
+
+#[cfg(feature = "costs_counting")]
+pub fn reset_evm_gas_counter() -> u64 {
```

Comment:
Is the global for gas counting a good idea?

Reply:
I wrapped it with EVM feature